### PR TITLE
Homogenize quickstart doc across all cloud platforms

### PR DESCRIPTION
**Pull request by fsismondi**

### CODE REVIEW

File: docs/modules/ROOT/pages/howtos/quickstart_k3s_libvirt.adoc [hunk 1]
```diff
@@ -22,8 +22,18 @@ mv terraform-provider-libvirt ~/.local/share/terraform/plugins/registry.terrafor
...
+Camptocamp's DevOps stack is a Terraform composition module
+has to be instantiated from the Terraform root module.
+
+Here is a minimal working example:
+
+
+=== Terraform root module
+
+```hcl
+# terraform/main.tf
```
Comment (mcanevet) [inline docs/modules/ROOT/pages/howtos/quickstart_k3s_libvirt.adoc:36]: the code block has no end

**Review by mcanevet — CHANGES_REQUESTED** [review]

**Review by mcanevet — DISMISSED** [review]

File: docs/modules/ROOT/pages/howtos/quickstart_aks.adoc [hunk 2]
```diff
@@ -10,13 +10,16 @@
 - Access to GitLab or GitHub (only supported CI/CD for now),
 - Knowledge of https://terraform.io[Terraform] basics
 
-== Create your Terraform root module
+== Create your Terraform files
 
 Camptocamp's DevOps stack is a Terraform composition module
-that has to be instantiated from the Terraform root module.
+has to be instantiated from the Terraform root module.
```
Comment (raphink) [inline docs/modules/ROOT/pages/howtos/quickstart_aks.adoc:16]: Seems to me that this was actually the correct version, and all others were wrong.

File: docs/modules/ROOT/pages/howtos/quickstart_aks.adoc [hunk 3]
```diff
@@ -10,13 +10,16 @@
 - Access to GitLab or GitHub (only supported CI/CD for now),
 - Knowledge of https://terraform.io[Terraform] basics
 
-== Create your Terraform root module
+== Create your Terraform files
```
Comment (raphink) [inline docs/modules/ROOT/pages/howtos/quickstart_aks.adoc:13]: What is the reason for not using `root module`?

File: docs/modules/ROOT/pages/howtos/quickstart_aks.adoc (hunk 2 above)
Comment (fsismondi) [reply docs/modules/ROOT/pages/howtos/quickstart_aks.adoc]: updated in latest commit

**Pull request closed without merging**